schema/ukrdc/UKRDC.xsd

```diff
@@ -1,4 +1,4 @@
-<xs:schema xmlns="http://www.rixg.org.uk/" xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.rixg.org.uk/" version="3.3.0">
+<xs:schema xmlns="http://www.rixg.org.uk/" xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.rixg.org.uk/" version="3.3.0-post1">
```

Comment:
Is this right? the -post1 was needed because our packages already had a 3.3.0 but the schema itself is 3.3.0?

I think I'd be happier if the version was changed to 3.3.1 just so the schema version itself stays in the same format.

Reply:
Agreed. This was only really added as an example. I'll update to 3.3.1 now on this PR. I think the point is more that it lets us include extra build info as defined in the semver standards that can be used everywhere.